tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -245,112 +304,263 @@ describe('Lobby - P1 Perspective', () => {
     // Disconnect socket and then opponent hits ready to start game
     cy.window()
       .its('cuttle.authStore')
       .then((store) => store.disconnectSocket());
     cy.readyOpponent();
-
     cy.drawCardOpponent();
-
     // Reload the page -- should bring user into the game
     cy.reload();
     assertGameStarted(false); // skip hand size assertion
     cy.get('[data-player-hand-card]').should('have.length', 6);
     cy.get('[data-opponent-hand-card]').should('have.length', 6);
   });
 });
+
+
+// helper functions
+
+function setup(isRanked = false) {
+  cy.wipeDatabase();
+  cy.visit('/');
+  cy.signupPlayer(myUser);
+  cy.createGamePlayer({ gameName: 'Test Game', isRanked }).then((gameSummary) => {
+    cy.window()
+      .its('cuttle.gameStore')
+      .then((store) => store.requestSubscribe(gameSummary.gameId));
+    cy.vueRoute(`/lobby/${gameSummary.gameId}`);
+    cy.wrap(gameSummary).as('gameSummary');
+  });
+}
+
+function playersArrayShouldBeEqualTo(length) {
+  cy.window()
+    .its('cuttle.gameStore')
+    .then((gameData) => {
+      expect(gameData.players.length).to.eq(length);
+    });
+}
+
+function checkPageContentOfLobby() {
+  // Displays title, game name and nav drawer
+    cy.contains('h1', en.lobby.lobbyFor);
+    cy.window()
+      .its('cuttle.gameStore')
+      .then((gameData) => {
+        cy.contains('h5', gameData.name);
+      });
+    cy.get('[data-cy=nav-drawer]').should('not.exist');
+  // Shows exit button, ready buttonn casual/ranked switch
+    cy.contains('button.v-btn', en.lobby.exit);
+    cy.contains('button.v-btn', en.lobby.ready);
+    cy.get('[data-cy=edit-game-ranked-switch]').should('exist');
+}
+
+function checkRanked(isRanked) {
+  if(isRanked) {
+    cy.get('[data-cy=ready-button-coffee-icon]').should('not.exist');
+    cy.get('[data-cy=ready-button-sword-cross-icon]').should('exist');
+    cy.contains('[data-cy=edit-game-ranked-switch]', en.global.ranked);
+  } else {
+    cy.get('[data-cy=ready-button-coffee-icon]').should('exist');
+    cy.get('[data-cy=ready-button-sword-cross-icon]').should('not.exist');
+    cy.contains('[data-cy=edit-game-ranked-switch]', en.global.casual);
+  }
+
+  cy.window()
+  .its('cuttle.gameStore')
+    .then((game) => {
+      expect(game.isRanked).to.eq(isRanked);
+    });
+}
+
+function assertGameStarted(noMovesYet = true) {
+  cy.url().should('include', '/game');
+  cy.window()
+    .its('cuttle.gameStore')
+    .then((game) => {
+      expect(game.players[1].hand.length).to.eq(6);
+      if (noMovesYet) {
+        expect(game.players.length).to.eq(2);
+        expect(game.players[0].hand.length).to.eq(5);
+        expect(game.deck.length).to.eq(39);
+      }
+      expect(game.topCard.rank).to.be.greaterThan(0);
+      expect(game.secondCard.rank).to.be.greaterThan(0);
+      expect(game.scrap.length).to.eq(0);
+      expect(game.twos.length).to.eq(0);
+    });
+}
+
+function clickReadyButton() {
+  cy.get('[data-cy=ready-button]').click();
+}
```

Comment:
I'm not sure this is worth putting into a helper function, it's really only one line and reading it in the test makes me think that it is doing something more than just clicking the button.